lib/Manager.php

```diff
@@ -247,10 +247,10 @@ public function getAnnouncement($id, $parseStrings = true, $ignorePermissions =
 	 */
 	public function getAnnouncements($limit = 15, $offset = 0, $parseStrings = true) {
 		$query = $this->connection->getQueryBuilder();
-		$query->select('a.*')
+		$query->select('a.announcement_id', 'a.announcement_time', 'a.announcement_user', 'a.announcement_subject', 'a.announcement_message', 'a.allow_comments')
```

Comment:
This can be reverted

Reply:
Can you test using:
```php
		$query->select('a.announcement_id')
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('allow_comments') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('announcement_time') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('announcement_user') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('announcement_subject') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('announcement_subject') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('announcement_message') . ')'))
			->addSelect($query->createFunction('ANY_VALUE(' . $query->getColumnName('allow_comments') . ')'))
```
instead of this line here and remove the change on groupBy

Reply:
Unfortunately this is not working for us as we are using a MariaDB which is pretty similar to a MySQL but hasn't implemented the `ANY_VALUE` function (see [MDEV-10426](https://jira.mariadb.org/browse/MDEV-10426)).

Reply:
arg, thats odd